contracts/croncat-tasks/src/helpers.rs

```diff
@@ -29,7 +32,7 @@ pub(crate) fn validate_boundary(
             }
         }
         (
-            Interval::Block(_) | Interval::Once | Interval::Immediate,
+            Interval::Block(_) | Interval::Once | Interval::Immediate | Interval::Cron(_),
```

Comment:
```suggestion
            _,
```
Since it's for any `Interval` now :thinking:. Unless we plan to add other interval later on

Reply:
Good one. I want to take a closer look at `Interval::Immediate` elsewhere in the code. Need to remember what use case differentiates `Immediate` and `Once` when there's a boundary involved.

Like if I say, "do this task at the boundary start of 5 blocks from now" it feels like both `Immediate` and `Once` will behave similarly. Perhaps the only difference is `Immediate` would technically be recurring, and fire every block after the boundary start and not after boundary end. Need to investigate a bit to confirm.